src/main/java/edu/harvard/iq/dataverse/AuxiliaryFile.java

```diff
@@ -1,20 +1,39 @@
 
 package edu.harvard.iq.dataverse;
 
+import edu.harvard.iq.dataverse.util.BundleUtil;
 import java.io.Serializable;
+import java.util.MissingResourceException;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
+import javax.persistence.NamedNativeQueries;
+import javax.persistence.NamedNativeQuery;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
 
 /**
  *
  * @author ekraffmiller 
  * Represents a generic file that is associated with a dataFile.
  * This is a data representation of a physical file in StorageIO
  */
+@NamedQueries({
+    @NamedQuery(name = "AuxiliaryFile.lookupAuxiliaryFile",
+            query = "select object(o) from AuxiliaryFile as o where o.dataFile.id = :dataFileId and o.formatTag = :formatTag and o.formatVersion = :formatVersion"),
+    @NamedQuery(name = "AuxiliaryFile.findAuxiliaryFiles",
+            query = "select object(o) from AuxiliaryFile as o where o.dataFile.id = :dataFileId"),
+    @NamedQuery(name = "AuxiliaryFile.findAuxiliaryFilesByType",
+            query = "select object(o) from AuxiliaryFile as o where o.dataFile.id = :dataFileId and o.type like :type"),
```

Comment:
Just noticed this - why "like" and not straight "="

Reply:
No reason. Fixed in e1d5541fd.